docs/tutorials/using-typeorm.md

```diff
@@ -105,41 +108,66 @@ If preferred, it is possible to define a `type` to clean up the transaction meth
 type TypeORMTransactionContext = TransactionContext<EntityManager>;
 ```
 
-### Unit Testing
-Use of TypeORM in the testing runtime is quite similar to using TypeORM in development, with the addition of appropriate placement of calls to set up and tear down the database schema.
+### Configuring TypeORM
 
-In `jest`, to set up the database once at the beginning of the test and tear down at the end:
-```javascript
-beforeAll(async () => {
-  testRuntime = await createTestingRuntime([DBOSAppClasses], "dbos-config.yaml");
-  await testRuntime.dropUserSchema(); // Optional
-  await testRuntime.createUserSchema();
-});
+:::info
+If you are using the [TypeORM template](#getting-started), this is done for you.
+:::
 
-afterAll(async () => {
-  await testRuntime.dropUserSchema();
-  await testRuntime.destroy();
-});
-```
+To enable TypeORM, you must set the `app_db_client` field in the [DBOS configuration file](../api-reference/configuration.md) to `typeorm`.
+You should also configure TypeORM migration commands.
+Here is an example of a configuration file set up for TypeORM:
 
-This will create the testing runtime, load TypeORM, and register the entities according to the decorators in `DBOSAppClasses`. The database will be configured according to the `database` section of the specified configuration `dbos-config.yaml` file:
 ```yaml
 database:
-  hostname: ${POSTGRES_HOST}
-  port: ${POSTGRES_PORT}
-  username: ${POSTGRES_USERNAME}
-  password: ${POSTGRES_PASSWORD}
-  app_db_name: ${POSTGRES_DATABASE}
-  system_database: 'opsys'
+  hostname: 'localhost'
+  port: 5432
+  username: 'postgres'
+  app_db_name: 'hello_typeorm'
+  password: ${PGPASSWORD}
   connectionTimeoutMillis: 3000
-  app_db_client: 'typeorm'
+  app_db_client: typeorm
+  migrate:
+    - npx typeorm migration:run -d dist/datasource.js
+  rollback:
+    - npx typeorm migration:revert -d dist/datasource.js
+runtimeConfig:
+  entrypoints:
+    - dist/src/operations.js
```

Comment:
Do we need these lines?

Reply:
Okay this wasn't very obvious. Because we added a "datasource.ts" file, now the compiled operations file is no longer under `dist/operations.js`

Reply:
The entities and migrations, which also need to be compiled, have the same effect.  We probably need to improve documentation around entrypoints later...